### ISSUE TITLE
unhandled page event *page.EventDownloadWillBegin

### ISSUE THREAD

**Issue by Amberlyd** [chromedp/chromedp#517]
#### What versions are you running?
The latest master branch

<pre>
$ go list -m github.com/chromedp/chromedp
The latest master
$ google-chrome --version
78.0.3904.108
$ go version
go 1.9.7
</pre>

#### What did you do? Include clear steps.
i wanted to download a file by clicking a button, but it throws an err.
i tryed to solve the issue writing "page.SetDownloadBehavior(page.SetDownloadBehaviorBehaviorAllow).WithDownloadPath("."),"but it didn't work

#### What did you expect to see?
download successful

#### What did you see instead?
err: unhandled page event *page.EventDownloadWillBegin

below is my code

```
func main() {
	dir, err := ioutil.TempDir("", "chromedp-example")
	if err != nil {
		panic(err)
	}
	defer os.RemoveAll(dir)

	headers := map[string]interface{}{
		"cookie": "session=d640ccf80c5859119524101d9905db15; csrf_token=7896178060",
	}

	opts := append(chromedp.DefaultExecAllocatorOptions[:],
		chromedp.DisableGPU,
		chromedp.NoDefaultBrowserCheck,
		chromedp.Flag("headless", false),
		chromedp.Flag("ignore-certificate-errors", true),
		chromedp.Flag("start-maximized", true),
		chromedp.UserDataDir(dir),
		chromedp.UserAgent("Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.108 Safari/537.36"),
	)

	allocCtx, cancel := chromedp.NewExecAllocator(context.Background(), opts...)
	defer cancel()

	// also set up a custom logger
	taskCtx, cancel := chromedp.NewContext(allocCtx, chromedp.WithLogf(log.Printf))
	defer cancel()

	// create a timeout
	taskCtx, cancel = context.WithTimeout(taskCtx, 20*time.Second)
	defer cancel()

	err = chromedp.Run(taskCtx,
		network.Enable(),
		network.SetExtraHTTPHeaders(network.Headers(headers)),
		chromedp.Navigate(`https://10.250.129.10/alarmReport.html?anchor=head&_id=3&type=active#history`),
		chromedp.WaitVisible(`body > footer`),
		page.SetDownloadBehavior(page.SetDownloadBehaviorBehaviorAllow).WithDownloadPath("."),
		chromedp.ActionFunc(func(context.Context) error {
			fmt.Printf("body finish\n")
			return nil
		}),
		chromedp.Click(`//span[@class="export-pdf"][1]`, chromedp.BySearch),
		chromedp.Sleep(5*time.Second),
	)
	if err != nil{
		log.Fatal(err)
	}
}
```

**Comment by mvdan** [comment]
Thanks, I was able to reproduce this with a simple program. I can fix the error; though I don't imagine it's causig any real bugs.

**Comment by Amberlyd** [comment]
Thanks for your reply. This bug has been driving me crazy these days. 
Waiting for your message!

**Comment by mvdan** [comment]
I forgot to link the commit to the issue, my bad. This was fixed in bbc198126faffcdca2ed78ecddccda3f14c668d0.
